upstream-community-operators/kong/kong.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,149 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
```

Comment:
Please add additional required fields to the CSV, as outlined in our docs: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md#required-fields-for-operatorhub